src/backend/api/reviews.js

```diff
@@ -1,49 +1,39 @@
 const express = require('express');
```

Comment:
Double check your queries, it seems you've mixed some code around, this file is for reviews but talks about data from the reservation table